src/index.ts

```diff
@@ -1,14 +1,22 @@
 import express from "express";
 import dotenv from "dotenv";
+import { logger } from "./services";
+import { connectDB } from "./config";
+import { router } from "./router";
 dotenv.config();
+connectDB();
 
 const port = process.env.PORT || 8000;
 const app = express();
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+
+app.use("/", router);
```

Comment:
According to the spec
```suggestion
app.use("/api/v1", router);
```